SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -101,17 +101,16 @@ struct ConfigureConnection
 		float TimeoutSeconds;
 #if WITH_EDITOR
 		TimeoutSeconds = Settings->HeartbeatTimeoutWithEditorSeconds;
-		
+
 #else
 		TimeoutSeconds = Settings->HeartbeatTimeoutSeconds;
 #endif
-		Worker_HeartbeatParameters WorkerHeartbeatParams = { static_cast<uint64_t>(Settings->HeartbeatIntervalSeconds
-																							   * SECONDS_TO_MILLIS),
-																	static_cast<uint64_t>(TimeoutSeconds * SECONDS_TO_MILLIS) };
-		Params.network.tcp.downstream_heartbeat = &WorkerHeartbeatParams;
-		Params.network.tcp.upstream_heartbeat = &WorkerHeartbeatParams;
-		Params.network.kcp.downstream_heartbeat = &WorkerHeartbeatParams;
-		Params.network.kcp.upstream_heartbeat = &WorkerHeartbeatParams;
+		Worker_HeartbeatParameters HeartbeatParams = { static_cast<uint64_t>(Settings->HeartbeatIntervalSeconds * 1000),
```

Comment:
To my first comment, this object is on the stack and will be destroyed when the function returns, so needs to remain a member variable.

Reply:
Oh I see, I misunderstood the first time sorry